components/director/internal/domain/application/resolver.go

```diff
@@ -267,6 +269,8 @@ func (r *Resolver) RegisterApplication(ctx context.Context, in graphql.Applicati
 
 	ctx = persistence.SaveToContext(ctx, tx)
 
+	log.Infof("Registering Application with name %s", in.Name)
```

Comment:
`Registering Application with name %s", in.Name` here vs `Application with id %s successfully registered", id` below. Be consistent in related logs. If at the first log location you don't have the id yet, atleast log also the name on the second location